build.gradle

```diff
@@ -159,6 +161,26 @@ tasks.withType(Test) {
     }
 }
 
+eclipse {
+	// re-generate APT configuration on Gradle project synchronisation
+	synchronizationTasks eclipseJdtApt, eclipseJdt, eclipseFactorypath
+
+	jdt.apt {
+        genSrcDir = file('bin/generated/sources/annotationProcessor/java/main')
```

Comment:
❔  just curious, why switch from build to bin?

Reply:
There's a convention of keeping eclipse and gradle build outputs separate. Bin is the eclipse path, we don't need to add the gradle path (in build), since classpath is an eclipse toolchain thing.